sql/plsql/PlSqlParser.g4

```diff
@@ -6614,6 +6872,7 @@ non_reserved_keywords_pre12c
     | SWITCHOVER
     | SYNCHRONOUS
     | SYNC
+    | SYS
```

Comment:
Not only `MEMOPTIMIZE` and `SYS`, but all new tokens from this pull request. They can be identifiers as well.